scss/login.scss

```diff
@@ -0,0 +1,75 @@
+.login-main {
+  align-items: center;
+  background-color: $white;
+  display: flex;
+  height: 100vh;
```

Comment:
I think that `100vh` is not necessary. Maybe you can use 100%

Reply:
Done